Classes/PhpBench/FrameworkEnabledBenchmark.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * (c) Contributors of the Neos Project - www.neos.io
+ * Please see the LICENSE file which was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\BuildEssentials\PhpBench;
+
+use Neos\BuildEssentials\TestableFramework;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Testing\RequestHandler\RuntimeSequenceInvokingRequestHandler;
+
+/**
+ * Tools for https://github.com/phpbench/phpbench
+ */
+class FrameworkEnabledBenchmark
+{
+    protected Bootstrap $flowBootstrap;
+
+    /**
+     * Use with phpbench BeforeClassMethods annotation to prepare persistence schema
+     */
+    public static function enablePersistence(): void
+    {
+        $_SERVER['FLOW_ROOTPATH'] = TestableFramework::calculateRootPath();
+        $bootstrap = TestableFramework::buildBootstrapWithPreselectedRequestHandler(TestableFramework::getApplicationContext(), RuntimeSequenceInvokingRequestHandler::class);
+        $bootstrap->run();
+        TestableFramework::enablePersistence($bootstrap);
+    }
+
+    /**
+     * Use with phpbench AfterClassMethods annotation to cleanup persistence schema
+     */
+    public static function cleanUpPersistence(): void
+    {
+        $_SERVER['FLOW_ROOTPATH'] = TestableFramework::calculateRootPath();
```

Comment:
```suggestion
        $this->withRootPath();
```

Reply:
Can't do because the function is static same reason as above.